src/renderer/src/routes/+page.svelte

```diff
@@ -25,6 +25,7 @@
 
 		return lairSetupRequired.subscribe((setupData) => {
 			if (setupData.isError) {
+				console.error(setupData.error);
```

Comment:
do we need this console.error?

Reply:
Yes please because it shows the stack trace as well, not just the message.